docs/index.md

```diff
@@ -0,0 +1,34 @@
+## Helm3 Charts for Sonatype Products
+### AWS Single-Instance Resilient Nexus Repository Manager Chart
+These charts will deploy a Nexus Repository Manager instance to an AWS EKS cluster.
+
+The current release has been tested on AWS EKS running Kubernetes version 1.21
+
+### Single-Instance Nexus Repository Manager OSS/Pro Kubernetes Chart
+These charts are designed to work out of the box with minikube using both Ingress
+and Ingress DNS addons.
+
+The current releases have been tested on minikube v1.25.1 running Kubernetes v1.23.1.
+
+### Add the Sonatype Repo to Your Helm
+
+`helm repo add sonatype https://sonatype.github.io/helm3-charts/`
+
+### Install a Server
+
+- Single-Instance Nexus Repository Manager OSS/Pro: `helm install nexus-repo sonatype/nexus-repository-manager`
+- AWS Single-Instance Resilient Nexus Repository Manager: `helm install nxrm sonatype/nxrm-aws-resiliency --values values.yaml`
```

Comment:
does the inclusion of values.yaml on this one, but not the others imply that there is _no_ useful default install?

Reply:
Yep, because it's AWS they'll be environment specific parameters which the user will need to provide e.g. subnet ids, arns for RDS credentials, license and so on